app/pages/search/search-controls/SearchControls.spec.js

```diff
@@ -0,0 +1,129 @@
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import queryString from 'query-string';
+import React from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import { browserHistory } from 'react-router';
+import simple from 'simple-mock';
+
+import SearchControls from './SearchControls';
+
+describe('pages/search/search-controls/SearchControls', () => {
+  function getWrapper(props) {
+    const defaults = {
+      initialValues: { search: '' },
+    };
+    return shallow(<SearchControls {...defaults} {...props} />);
+  }
+
+  describe('render', () => {
+    it('renders form with correct onSubmit', () => {
+      const wrapper = getWrapper();
+      const form = wrapper.find('form');
+      expect(form).to.have.length(1);
+      expect(form.prop('onSubmit')).to.equal(wrapper.instance().handleSearch);
+    });
+
+    describe('search control', () => {
+      const search = 'search text';
+      let wrapper;
+      let searchContorlWrapper;
+
+      before(() => {
+        wrapper = getWrapper({ initialValues: { search } });
+        searchContorlWrapper = wrapper.find('[controlId="search-control-group"]');
```

Comment:
- [ ] searchCont**orl**Wrapper -> searchCont**rol**Wrapper
- Instead of using `before` for these kinds of things, I'd rather define new get...Wrapper functions if it's simple to do. So for example these tests would be

  ```js
  function getSearchControlWrapper(values) {
    const wrapper = getWrapper({ initialValues: { search: 'search text', ...values } });
    return wrapper.find('[...]');
  }
  it('has correct label', () => {
    const controlLabel = getSearchControlWrapper().find(ControlLabel);
  ```

  That way makes it much easier to add test cases for different params. Not important to change this, though.